### DISCUSSION TITLE
Toggle component

### DISCUSSION THREAD

**Discussion by jonhendershot** [twilio-labs/paste#2216] [Ideas] [answered]
Hey there! Did this component ever make it in to Paste? I see that it's referenced on the [Checkbox docs](https://paste.twilio.design/components/checkbox/#about-checkbox) as well, but don't see it in Paste docs or roadmap. 

---

Hi @sang-yeo,

According to our roadmap, it should be expected around Q4 2021.  It's under the `2021.11.16` release on our [roadmap page](https://paste.twilio.design/roadmap).  Keep in mind that the further out we are from the release time-wise, the less reliable the specific dates become.  

Hope this helps!

_Originally posted by @TheSisb in https://github.com/twilio-labs/paste/discussions/1592#discussioncomment-906572_

**Comment by TheSisb** [comment]
Hi @JonHendershot! We're undergoing roadmap reviews at the moment, which determine how we prioritize work on upcoming components like the toggle/switch component. We'll have more information about this when the process concludes in a few weeks time. In the meantime, if a toggle/switch is something that's needed urgently, it's possible to make it from the Box primitive.

We'll update this when there's more information. Many thanks!

**Comment by SiTaggart** [comment, marked as answer]
Hey @JonHendershot 

Just a quick note to say the Switch Component just landed in Paste!

https://paste.twilio.design/components/switch/

Can't wait to see this in Prod!
